City-of-Helsinki/kukkuu-admin
# KK-633, KK-634, KK-629 | Mock event group details

## Description

Contains various small additions that make it possible to implement the referenced tickets faster.

* Adds `MutationConfirmationButton` that was used to implement the publish button for event groups
* Adds `LocalDataGrid` which is a barebones version of `DataGrid` that can be given lists of records to render
* Refactors `Breadcrumb` component so that it allows for more complex breadcrumbs to be built

## Context

[KK-633](https://helsinkisolutionoffice.atlassian.net/browse/KK-633)
[KK-634](https://helsinkisolutionoffice.atlassian.net/browse/KK-634)
[KK-629](https://helsinkisolutionoffice.atlassian.net/browse/KK-629)

## How Has This Been Tested?

I've added unit/integration level tests for the new components (`MutationConfirmationButton`, `LocalDataGrid`, `Breadcrumb`)

## Manual Testing Instructions for Reviewers

There's no real data yet, but I've changed the API to return mock data, which allows the show view to work.

1. Access event group show-view /event-groups/1/show
1. Expect to see the correct breadcrumb
1. Expect to see a confirmation modal when clicking publish button
1. Expect to see a list of "events"
